dashboard/config/linux/bits/arm64.yml

```diff
@@ -2,9 +2,9 @@
 # Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
 shell:
- # android-5.4.yml has own defconfig.
- - make defconfig: [-android]
- - make kvm_guest.config: [-android]
+ # android-5.4.yml and cos-5.15.yml have their own defconfig.
+ - make defconfig: [-android, -cos]
```

Comment:
I see we have `nodefconfig` tag and most andoid instances already use it.
It looks reasonable to replace a custom set of tags here with `nodefconfig`.